demisto_sdk/commands/lint/tests/test_linter/docker_runner_test.py

```diff
@@ -84,9 +84,13 @@ class TestPytest:
                              argvalues=[(0, 0),
                                         (1, 1),
                                         (2, 1),
-                                        (5, 0)])
+                                        (5, 0),
+                                        (137, 1),
+                                        (139, 1),
+                                        (143, 1),
+                                        (126, 1)])
     def test_run_pytest(self, mocker, linter_obj: Linter, exp_container_exit_code: int, exp_exit_code: int):
-        exp_test_json = mocker.MagicMock()
+        exp_test_json = mocker.MagicMock() if exp_container_exit_code in [0, 1, 2, 5] else {}
```

Comment:
Why does it behave differently?

Reply:
`test_json = json.loads(get_file_from_container(container_obj=container_obj,
                                                               container_path="/devwork/report_pytest.json",
                                                               encoding="utf-8"))`

this line is used in error codes 0-5, to get the pytest report and parse it. Do we need the same behaviour in container exit codes?